### DISCUSSION TITLE
Semester 4 Week 8

### DISCUSSION THREAD

**Discussion by Stev3eM** [SE-TINF22B6/CookHub#289] [Show and tell]
Hello backpackers,

New updates out now!

<br>

## Frontend

✅ Finished liking a recipe functionality

✅ Connected endpoint to change username

✅ Made pages responsive
- Settings
- Profile
- FindRecipe
- MyRecipes

<img src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/70bf10f5-bc3a-443a-8a99-ef0f660a1ef4" height="300" />
<img src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/6cdf1d31-1111-4ef3-bc85-015ac6971084" height="300" />
<img src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/58e3bd70-38fe-425b-b10a-b58775655da1" height="300" />
<img src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/7dc87f6a-2b51-4490-9a10-cb2f47b4f1ac" height="300" />

<br>
<br>

## Backend

✅ Done Requests for Account Settings Page
- change userName
- change password
- delete Account
- change ProfilePicture

✅ Some Backend Refactoring
- extracted methods for test data generation

✅ Created ViewRecipe Endpoints

<br>
<br>

## Testing

✅ Tests for new Account Requests (xUnit .NET)

✅ Marc has written hundreds of millions of Selenium tests (20)

![image](https://github.com/SE-TINF22B6/CookHub/assets/113697513/b6606695-cf98-452b-bb94-8c24bf5af4b6)

<br>
<br>

## Retroperspective

Summary = { ∅, 

<img src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/10fe1cc3-a0f3-4dc9-8987-411335e5488c" height="800" />

<br>
<br>

- We've retroperspectived a lot about different technolgies and which we absolutely never will use again 
- We've learned to use a Marc (the ancient one) in future projects
- We've specified the priorities to focus on during the final project phase

<br>

}

<br>
<br>

## Team Building

✅ Pair Programming

<br>
<br>

See you next time!

**Your**   <img width="50" alt="image" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/1a218c37-d3b0-4c88-9ae1-827b5a4eba42">   **Team**

**Comment by Nahlam4** [comment]
Dear Team CookHub, 
nice job this week, especially with your tests and your frontend 😄 
Personally, i think that your font for your headers is a bit confusing and if the time allows for it, i would change it to something simpler. 
Other than that i like your progress, keep it up now for the last week 💯 - your Team DHBWhub
